vnext/Microsoft.ReactNative/Modules/NativeUIManager.cpp

```diff
@@ -69,13 +92,63 @@ static int YogaLog(
 }
 #endif
 
+class LayoutConformanceVisitor : public ShadowNodeVisitor {
+  using Super = ShadowNodeVisitor;
+
+ public:
+  LayoutConformanceVisitor(YGErrata errata) : m_errata{errata} {}
+
+ protected:
+  void VisitCore(ShadowNodeBase *node) override {
+    if (!node->m_layoutConformance.has_value()) {
+      ApplyYogaConfig(node);
+      Super::VisitCore(node);
```

Comment:
nit: Consider making `ShadowNodeVisitor` pure virtual, then having `VisitCore` call the implemented virtual method during traversal, instead of needing to explicitly re-call the same method.

e.g. `boolean Apply(ShadowNodeBase *node)`, where return is whether to keep propagating.

Reply:
I abstracted this from the existing TextNodeVisitor, and I'm trying to minimize churn to TextNodeVisitor. Seems like a good thing to do in a future cleanup diff.